src/chrome/content/rules/1und1.de.xml

```diff
@@ -17,14 +10,20 @@ Fetch error: http://tt.1und1.de/ => https://tt.1und1.de/: (6, 'Could not resolve
 
 	Nonfunctional subdomains:
 
+		- apps ⁴
+		- img ⁵
+		- login ⁴
 		- mirror ¹
 		- newsroom ²
 		- presse ²
+		- tt ⁵
 		- tv ³
```

Comment:
`erfolgscenter`
`service-center`

Reply:
Could you add them as comments since they have "Complications"?

Reply:
They are already added as comments at lines 29 and 31.